aspnetcore/migration/50-to-60/samples/WebRP31to60/Program.cs

```diff
@@ -0,0 +1,63 @@
+//#define V60 // V31  V60 V60A // Defined in project file
+#if NEVER
+#elif V31
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+#region snippet
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureWebHostDefaults(webBuilder =>
+            {
+                webBuilder.UseStartup<Startup>();
+            });
+}
+#endregion
+#elif V60
+#region snippet_60
+using Microsoft.AspNetCore.Builder;
+
+var builder = WebApplication.CreateBuilder(args);
+
+var startup = new Startup(builder.Configuration);
```

Comment:
I really like how you show to use existing Startup classes with WebApplication manually 👍